code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -30,6 +30,7 @@
 	if(occupant)
 		vis_contents -= occupant
 		REMOVE_TRAIT(occupant, TRAIT_IMMOBILIZED, CRYO_TRAIT)
+		REMOVE_TRAIT(occupant, TRAIT_FORCED_STANDING, CRYO_TRAIT)
 		if(occupant.resting || HAS_TRAIT(occupant, TRAIT_FLOORED))
 			occupant.set_lying_down()
```

Comment:
```suggestion
		REMOVE_TRAIT(occupant, TRAIT_FORCED_STANDING, CRYO_TRAIT)
```
And this one is processed when the trait is properly removed.

Reply:
....and done!